packages/dd-trace/src/config.js

```diff
@@ -449,10 +354,25 @@ class Config {
     }
 
     // TODO: test
-    this._applyCalculated()
+    this._applyCalculated(options)
```

Comment:
here you call applyCalculated with options but the applyCalculated function don't even have arguments in this PR ?